.vscode/settings.json

```diff
@@ -24,4 +24,5 @@
   "javascript.preferences.importModuleSpecifier": "relative",
   "typescript.preferences.importModuleSpecifier": "relative",
   "prettier.configPath": ".prettierrc.js",
+  "editor.formatOnSave": true,
```

Comment:
No, we don't use this option! This is a deprecated feature. We below rather use in our local project.

```
"editor.codeActionsOnSave": {
    "source.fixAll": true
  },
```

Reply:
I highly recommend installing the extension **`ESLint`** on VScode and set the formatting. It will prevent from many possible futures bugs.

Then,
Mac) Open command pallette:  `Cmd` + `Shift` + `P`

<img width="1920" alt="스크린샷 2020-07-15 19 34 08(2)" src="https://user-images.githubusercontent.com/50701501/87535373-5afa3400-c6d2-11ea-9f4c-4b2a0d70e504.png">

Reply:
Thank you for your kindness. I've already installed lint but I think something mixed up and got wrong setting. I was silly! I changed setting and It will be okay from now. Cheers!